src/model/LabyrinthList.java

```diff
@@ -1,81 +0,0 @@
-package model;
-
-import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthList extends LabyrinthImpl {
-
-	private HashMap<Position, Integer> list;
-
-	public LabyrinthList(String path) {
-		try {
-			Scanner sc = new Scanner(new File(path));
-			startValue = sc.nextInt();
```

Comment:
reading startValue, finishValue, freeValue, wallValue is common code with matrix version I would consider to have this part extracted into a method in abtract  super class.